presentation/build.gradle

```diff
@@ -46,4 +46,5 @@ dependencies {
     testImplementation TestDependencies.junit
     androidTestImplementation TestDependencies.testJunit
     androidTestImplementation TestDependencies.espressoCore
-}
+    implementation 'com.google.android.material:material:1.0.0'
```

Comment:
Put this above the test implementations.

Reply:
done